hydromt_wflow/wflow_sediment.py

```diff
@@ -647,7 +653,8 @@ def setup_lulcmaps_from_vector(
             * Optional variable: ["forest_type"]
         lulc_vars : dict
             List of landuse parameters to prepare.
-            By default ["landuse","Kext","Sl","Swood","USLE_C","PathFrac"]
+            By default ["landuse","vegetation_kext","vegetation_leaf_storage",
+            "vegetation_wood_storage","USLE_C","soil_compacted_fraction"]
```

Comment:
I think I forgot to update the docstrings here. It's now a dict that you can copy from the function setup_lulcmaps above